axelor-account/src/main/java/com/axelor/apps/account/exception/IExceptionMessage.java

```diff
@@ -171,6 +171,8 @@
   static final String MOVE_9 = /*$$(*/ "Tax is mandatory for the account %s" /*)*/;
   static final String MOVE_10 = /*$$(*/
       "Analytic distribution template is mandatory for the account %s." /*)*/;
+  static final String MOVE_11 = /*$$(*/
```

Comment:
A lot of constant in this interface are named following the pattern "{classname}_{number}" but there is no real reason to name it that way. It's an error from old legacy code. So for new constants, it's better to use a meaningful name instead of MOVE_11.